app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -101,7 +101,7 @@ $white: #ffffff;
 // $vapor       : #F6F6F6;
 // $white-smoke : #F5F5F5;
 // $silver      : #EFEFEF;
-// $smoke       : #EEEEEE;
+$smoke       : #EEEEEE;
```

Comment:
Color `#EEEEEE` should be written as `#eeeeee`<br>Variable names should be followed immediately by a colon